core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/edit-ema-toolbar/edit-ema-toolbar.component.ts

```diff
@@ -127,7 +125,7 @@ export class EditEmaToolbarComponent {
      */
     onPersonaSelected(persona: DotPersona & { pageId: string }) {
         if (persona.identifier === DEFAULT_PERSONA.identifier || persona.personalized) {
-            this.updateQueryParams({
+            this.uveStore.updatePageParams({
                 'com.dotmarketing.persona.id': persona.identifier
```

Comment:
I think we talked about renaming this queryParam at some point, we could include it in this PR?

Reply:
Now it's `pageParams`